packages/ra-ui-materialui/src/layout/LinearProgressIfTooLong.tsx

```diff
@@ -0,0 +1,9 @@
+import { useTimeout } from 'ra-core';
+import * as React from 'react';
+import LinearProgress from './LinearProgress';
+
+export const LinearProgressIfTooLong = ({ timeout = 1000, ...props }) => {
```

Comment:
I think we don't need a second component for that. It should be the default behavior of our LinerProgress